std/python/Lib.hx

```diff
@@ -36,7 +36,7 @@ class Lib {
 	/**
 		Print the specified value on the default output.
 	**/
-	public static function print(v:Dynamic):Void {
+	public static function print_(v:Dynamic):Void {
```

Comment:
Then I can't use my PR, so there is no incentive for me to work on it anymore. Previous code was better for me, because indirection allowed to keep `print` name in API, but use `print_` in generated, where it does't matter.